.reek.yml

```diff
@@ -0,0 +1,8 @@
+detectors:
+  DuplicateMethodCall:
+    max_calls: 3
+  IrresponsibleModule:
+    enabled: false
+
+exclude_paths:
+  - vendor/
```

Comment:
Reek tries to dig through the `./vendor` in CI (where gems are installed), so… that's why this is here.